recipes/openjdk/all/conanfile.py

```diff
@@ -0,0 +1,51 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class OpenJDK(ConanFile):
+    name = "openjdk"
+    url = "https://github.com/conan-io/conan-center-index/"
+    description = "Java Development Kit builds, from Oracle"
+    homepage = "https://jdk.java.net"
+    license = "GPL-2.0-with-classpath-exception"
+    topics = ("java", "jdk", "openjdk")
+    settings = "os", "arch"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if self.settings.arch != "x86_64":
+            raise ConanInvalidConfiguration("Unsupported Architecture.  This package currently only supports x86_64.")
+        if self.settings.os not in ["Windows", "Macos", "Linux"]:
+            raise ConanInvalidConfiguration("Unsupported os. This package currently only support Linux/Macos/Windows")
+
+    def source(self):
```

Comment:
```suggestion
    def build(self):
```

Pre-built should be downloaded to build folder directly: https://docs.conan.io/en/latest/creating_packages/existing_binaries.html#downloading-and-packaging-pre-built-binaries